gnome-initial-setup/gis-page-util.c

```diff
@@ -492,3 +491,94 @@ gis_page_util_show_demo_dialog (GisPage *page)
   gtk_window_set_modal (GTK_WINDOW (demo_dialog), TRUE);
   gtk_window_present (GTK_WINDOW (demo_dialog));
 }
+
+static gboolean
+run_dialog_cb (gpointer user_data)
+{
+  GtkDialog *dialog = GTK_DIALOG (user_data);
+  gtk_dialog_run (dialog);
+  return G_SOURCE_REMOVE;
+}
+
+static void
+on_reformatter_exited (GTask  *task,
+                       GError *error)
+{
+  GisPage *page = GIS_PAGE (g_task_get_source_object (task));
+  GisDriver *driver = page->driver;
+
+  gis_driver_show_window (driver);
+
+  if (error == NULL)
+    {
+      g_task_return_boolean (task, TRUE);
+    }
+  else
+    {
+      GtkWindow *toplevel = GTK_WINDOW (gtk_widget_get_toplevel (GTK_WIDGET (page)));
+      GtkWidget *message_dialog;
+
+      g_critical ("Error running the reformatter: %s", error->message);
+
+      message_dialog = gtk_message_dialog_new (toplevel,
+                                               GTK_DIALOG_MODAL | GTK_DIALOG_DESTROY_WITH_PARENT,
+                                               GTK_MESSAGE_ERROR,
+                                               GTK_BUTTONS_CLOSE,
+                                               _("Error running the reformatter: %s"), error->message);
```

Comment:
Missing `TRANSLATORS` comment. Maybe not so important if this is just for Endless, but still probably worth it?

Reply:
ok.